tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,41 @@
+"""Handles logging functionality."""
+
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from inspect import getframeinfo
+from inspect import stack
+
+def log(log_priority, publisher_id, log_payload, job_id=None,
+        flow_id=None, parent_id=None, cluster_id=None,
+        message_id=None, timestamp=None, node_id=None,
+        caller=None, *args, **kwargs):
+    """
+    Function used for logging errors/output/info.
+    Args:
+        log_priority [Type : String]: Priority of the Log Message (error/info)
+        publisher_id [Type : Integer] : Id of publisher (mandatory)
+        log_payload [Type: Dict] : Payload can contain /
+                                   parameters like message that is to be logged
+    """
+    if caller is None:
+            caller = getframeinfo(stack()[1][0])
+            caller = {"filename": caller.filename,
+                      "line_no": caller.lineno,
+                      "function": caller.function}
+    try:
+        if publisher_id is not None:
```

Comment:
you can add this check in https://github.com/rishubhjain/commons/blob/c230cd8ec64390af84120e1a71364085fd7069d8/tendrl/commons/message.py#L75

like
 if publisher_id is None:
    return False